samples/WebBluetoothGroveLcdDemo.js

```diff
@@ -186,7 +186,6 @@ ble.on('advertisingStart', function(error) {
 
 ble.on('accept', function(clientAddress) {
     console.log("Client connected: " + clientAddress);
-    glcd.clear();
```

Comment:
Why this change?

Reply:
@grgustaf If not remove glcd.clear() function, the Grove LCD will be cleared when another BLE client connected, and the Grove LCD will be not displayed until value updated.